src/DataFixtures/BrandFixtures.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use App\Entity\Brand;
+
+/**
+ * Class BrandFixtures
+ *
+ * @package App\DataFixtures
+ */
+class BrandFixtures extends Fixture
+{
+    const BRAND_REFERENCE = 'brandname';
+
+    /**
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager)
+    {
+        for ($i = 1; $i <= 100; $i++){
+        $brand = new Brand();
+        $brand ->setName('brand' .$i);
+        $this->setReference(self::BRAND_REFERENCE, (object)$brand);
```

Comment:
Créer 1 référence par marque, et pas 1 référence qui sera remplacée à chaque itération `self::BRAND_REFERENCE . $i`

Reply:
Modification effectuée voir le commit 27c3f7ce063d0f21f4b39b007f93fedbf6ce0e4f.